### PR TITLE
feat(playwright): Upgrade to axe-core@4.3.2

### PR DESCRIPTION
**Pull request by michael-siek**
~~blocked by: https://github.com/dequelabs/axe-core-npm/pull/335~~

### CODE REVIEW

**Review by WilcoFiers — CHANGES_REQUESTED** [review]

File: packages/playwright/src/index.ts [hunk 1]
```diff
@@ -161,4 +180,68 @@ export default class AxeBuilder {
...
+      let childResults: AxePartialRunner | null = null;
+      try {
+        const iframeHandle = await frame.evaluateHandle(shadowSelect, {
+          frameSelector
+        });
+        // note: these can return null but the catch will handle this properly for all cases
+        const iframeElement = iframeHandle.asElement();
+        const childFrame = await (
+          iframeElement as ElementHandle<Element>
+        ).contentFrame();
```
[the remark this answers is not in the document]
Comment (michael-siek) [reply packages/playwright/src/index.ts:232]: This is a typeof `Frame`

File: packages/playwright/src/index.ts [hunk 2]
```diff
@@ -114,22 +123,32 @@ export default class AxeBuilder {
    */
 
   public async analyze(): Promise<AxeResults> {
```
[the remark this answers is not in the document]
Comment (michael-siek) [reply packages/playwright/src/index.ts:125]: https://github.com/dequelabs/axe-devtools-npm/pull/1127#issue-596040288

**Review by straker — APPROVED** [review]
LGTM. Do we need Wilco to approve his one comment? https://github.com/dequelabs/axe-core-npm/pull/334/files#r690399615

**Review by straker — DISMISSED** [review]
Sorry, forgot I had a change marked

File: packages/playwright/src/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply packages/playwright/src/index.ts:125]: If we do need to change it, it should be in a separate PR. Don't let that hold this up.

**Review by straker — APPROVED** [review]

**Pull request merged**